### PR TITLE
VAGOV-4760

### PR DESCRIPTION
**Pull request by ethanteague**
## Description

- Front End work for outreach and assets 
- Here's the reader's digest version of what this pr does: move `/outreach-and-events?q=events` to `/outreach-and-events/events` and move `/outreach-and-events?q=benefits` to `/outreach-and-events/benefit-materials`. Doing so required us to setup two new content types in drupal. I added two new graphql queries, and two new templates. This was mostly just copy paste stuff, and visually, the appearances of the new paths are identical to the appearances of the old paths

## Testing done

- Visual

## QA
- [ ] See Jira - too be performed on stage by @jefflbrauer

### CODE REVIEW

**Comment by ncksllvn** [comment]
Woo, looks like this page is in [dev](https://dev.va.gov/outreach-and-events/index.html?q=benefits) which means it can be pulled up [locally](http://localhost:3001/outreach-and-events/?q=benefit). Maybe that's been true for awhile, but this is the first time I've seen it outside of Staging.

@ethanteague Is there any way for other FE engineers to pull this work up locally with having to login to Drupal? It would make things immensely easier and clearer for reviewers if they could see exactly what it is they're reviewing via links, screenshots, or ideally, both. I tend to add screenshots to every one of my [pull requests](https://github.com/department-of-veterans-affairs/vets-website/pull/10374) that contain visual changes. Doing these things would really help move things forward and get this work out of review and merged that much more quickly.

**Comment by ethanteague** [comment]
> Woo, looks like this page is in [dev](https://dev.va.gov/outreach-and-events/index.html?q=benefits) which means it can be pulled up [locally](http://localhost:3001/outreach-and-events/?q=benefit). Maybe that's been true for awhile, but this is the first time I've seen it outside of Staging.
> 
> @ethanteague Is there any way for other FE engineers to pull this work up locally with having to login to Drupal? It would make things immensely easier and clearer for reviewers if they could see exactly what it is they're reviewing via links, screenshots, or ideally, both. I tend to add screenshots to every one of my [pull requests](https://github.com/department-of-veterans-affairs/vets-website/pull/10374) that contain visual changes. Doing these things would really help move things forward and get this work out of review and merged that much more quickly.

I don't think you will be able to qa this without an instance of drupal running, @ncksllvn - I usually include screenshots for frontend specific stuff, but this is mostly drupal config that is required for qa purposes. Are you looking for drupal screenshots? Would it be easier for you to hop on a call and have me screenshare with you while you qa from my machine?

**Comment by jbalboni** [comment]
@ethanteague @ncksllvn One option would be for Ethan to do a local build, zip up the cache, and upload that somewhere where a dev could download it and use it locally.

**Comment by ethanteague** [comment]
> @ethanteague @ncksllvn One option would be for Ethan to do a local build, zip up the cache, and upload that somewhere where a dev could download it and use it locally.

This option will still require docker setup (with Lando) / configuration - might not be quick, but I'm happy to provide a zipped instance if yall wanna proceed with this method, @jbalboni @ncksllvn

**Comment by jbalboni** [comment]
@ethanteague I'm not sure I follow. Is that Docker set up you have to do locally? I meant the cache that's in `vets-website/.cache/localhost`, which shouldn't require anything running locally for a developer who drops that into their local repo clone.

**Comment by ethanteague** [comment]
> @ethanteague I'm not sure I follow. Is that Docker set up you have to do locally? I meant the cache that's in `vets-website/.cache/localhost`, which shouldn't require anything running locally for a developer who drops that into their local repo clone.

Sorry, @jbalboni - the docker setup is for the drupal instance so that a user can run through the config setup. The qa for this task is mostly related to making sure drupal user config works as expected, so that's why I'm thinking this might be better handled on stage.

File: src/site/layouts/event_listing.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        {% include "src/site/components/navigation-sidebar-trigger.html" with menuId = "va-detailpage-sidebar" %}
+        {% if !entityPublished %}
+        <div class="usa-alert usa-alert-info">
+          <div class="usa-alert-body">
+            <p class="usa-alert-text">You are viewing a draft.</p>
+          </div>
+        </div>
+        {% endif %}
+
+        <article class="vads-l-grid-container--full--full">
```
Comment (jbalboni) [inline src/site/layouts/event_listing.drupal.liquid]: This class seems to have an extra `--full`? There are several other instances of this.

Comment (ethanteague) [reply src/site/layouts/event_listing.drupal.liquid]: Good catch, @jbalboni - fixed.

**Review by ncksllvn — APPROVED** [review]
I'm approving this because I understand it's a blocker for your team and in return mine, but I want to stress that this is essentially a blind approval, because there's hardly any value in having another FE engineer review this when we have no known ability of actually running this code locally or visiting its result on a review instance, and most folks have a lot less context than I do about what this code actually does or what the Outreach and Events product is. I can read the templates and make suggestions based off that, but without true knowledge or insight (such as into the data structures or objectives of the work) we're just too limited. 

So again, I approve this, but if we want to see value come out of a code review and have code reviews happen in a timely manner, we have to consider what we can do to make things easier for other FE engineers to review.

**Pull request merged**